src/screens/HyperSwitch/UserManagement/HSwitchShowUserData.res

```diff
@@ -73,10 +73,17 @@ let make = () => {
 
   let getRoleForUser = async () => {
     try {
+      // TODO - Temp fix - Backend fix awaited
       let url = getURL(
         ~entityName=USER_MANAGEMENT,
         ~userRoleTypes=ROLE_ID,
-        ~id=Some(currentSelectedUser.role_id),
+        ~id={
+          Some(
+            currentSelectedUser.role_id === "org_admin"
```

Comment:
please recheck this flow

Reply:
Tested Jeeva. It's a temporary fix. Will remove when backend will be fixed